src/ocrd_network/client.py

```diff
@@ -1,28 +1,62 @@
-from json import dumps, loads
-from requests import post as requests_post
 from ocrd_utils import config, getLogger, LOG_FORMAT
-
-from .constants import NETWORK_PROTOCOLS
+from .client_utils import (
+    get_ps_deployed_processors,
+    get_ps_deployed_processor_ocrd_tool,
+    get_ps_processing_job_log,
+    get_ps_processing_job_status,
+    get_ps_workflow_job_status,
+    poll_job_status_till_timeout_fail_or_success,
+    poll_wf_status_till_timeout_fail_or_success,
+    post_ps_processing_request,
+    post_ps_workflow_request,
+    verify_server_protocol
+)
 
 
 class Client:
-    def __init__(self, server_addr_processing: str = config.OCRD_NETWORK_SERVER_ADDR_PROCESSING):
+    def __init__(
+        self,
+        server_addr_processing: str,
```

Comment:
```suggestion
        server_addr_processing: Optional[str],
```

But why not assign the default directly for `server_addr_processing` but do so for `timeout` and `wait`?

Reply:
Because I did not want to place defaults for each click option `--address` for all cli methods. Since the address is passed directly to the constructor without checking if set or not, it overwrites the `server_addr_processing` with `None`. Since there are no `timeout` and `wait` flags in the client cli, these are set by default.